src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -169,10 +170,10 @@ public static void WebSocket_Http_Duplex_BinaryStreamed()
         }
     }
 
-    [Fact]
+    [ConditionalFact(nameof(WebSockets_Supported), nameof(Root_Certificate_Installed))]
     [OuterLoop]
 #if !FEATURE_NETNATIVE
-    [ActiveIssue(420, PlatformID.AnyUnix)]
+    //[ActiveIssue(420, PlatformID.AnyUnix)]
```

Comment:
Then the condition is really whether it's Windows instead of whether websocket is supported. I think you are right "this gets to the heart of what [ConditionalFact] means". I created #1034 to track this discussion. I prefer we don't change websocket tests just yet, but I will not block on this.